tests/wpt/mozilla/tests/mozilla/duplicated_scroll_ids_ref.html

```diff
@@ -3,9 +3,17 @@
     <head>
         <meta charset="utf-8">
         <title>Ensure that content which produces duplicate scroll ids does not panic</title>
+        <style>
+            #details {
+                display: list-item;
+                counter-increment: list-item 0;
+                list-style: disclosure-open inside;
+            }
+        </style>
     </head>
     <body>
         <div style="width: 100px; height: 100px; background: green"></div>
+        <div id="details">Details</div>
```

Comment:
Is this change related to the duplicate scroll ids test here?

Reply:
I think you can probably just remove this test. It doesn't produce duplicated scroll ids any longer.